src/kube-runtime/src/init

```diff
@@ -78,6 +79,10 @@ PAI_RUNTIME_DIR=${PAI_WORK_DIR}/runtime.d
 
 PAI_LOG_DIR=${PAI_WORK_DIR}/logs/${FC_POD_UID}
 
+# Clean ${PAI_WORK_DIR} and ${PAI_LOG_DIR} since they may contain last execution content. (rarely happen, but seen in real world)
+rm -rf ${PAI_WORK_DIR}/*
+rm -rf ${PAI_LOG_DIR}/*
```

Comment:
do you need to clean log dir? keep the previous log may be helpful

Reply:
Sure, will change the code

---
In reply to: [332808965](https://github.com/microsoft/pai/pull/3695#discussion_r332808965) [](ancestors = 332808965)